npm/private/npm_translate_lock_state.bzl

```diff
@@ -53,6 +51,8 @@ WARNING: `update_pnpm_lock` attribute in `npm_translate_lock(name = "{rctx_name}
     if _should_update_pnpm_lock(priv):
         _init_importer_labels(priv, label_store)
 
+    _init_patch_labels(priv, rctx, label_store)
```

Comment:
The patch labels now have to be processed after parsing the lockfile since some of the labels now come from the lockfile.